adafruit_clue.py

```diff
@@ -64,12 +69,12 @@ class _ClueSimpleTextDisplay:
 
     def __init__(  # pylint: disable=too-many-arguments
         self,
-        title=None,
-        title_color=0xFFFFFF,
-        title_scale=1,
-        text_scale=1,
-        font=None,
-        colors=None,
+        title: str = None,
+        title_color: Union[int, Tuple] = 0xFFFFFF,
```

Comment:
`Tuple` is a generic, so you can add more information if you want!

Reply:
Here's where it looks like the more detailed information can be found:  https://github.com/adafruit/Adafruit_CircuitPython_Display_Text/blob/59c27577053005435bb5466d07befa04f8d4df2f/adafruit_display_text/label.py#L51

Reply:
Good suggestion, I updated it here and in the other occurencies as well.